manager/manifests/fluentd.yaml

```diff
@@ -168,10 +168,15 @@ metadata:
   name: fluentd
   namespace: default
 spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
   template:
     metadata:
       labels:
         app: fluentd
+        hallo: dummy
```

Comment:
This is probably left over from testing :)

Reply:
will correct